### DISCUSSION TITLE
What to do to add new language (greenlandish)?

### DISCUSSION THREAD

**Discussion by schris-dk** [globaleaks/globaleaks-whistleblowing-software#3360] [General]
If we decide to go for a tranlation of the text/setup in GL - what files do we need to translate - and can we add them to our current setup or will the eventually be part of an update?

**Comment by evilaliv3** [comment]
Hello @schris-dk 

Translations can be performed on a simple web platform that we use to coordinate translations efforts that is called Transifex.

If you could pass me the email of the translators for each needed languages we could send them an invite. It would be great to have a first translator and a reviewer so to raise the quality of the work. 
You could write directly to me on https://community.globaleaks.org for this.

To incentivate translations for our communities we commit to release an update including the translation as soon that the translation is available.

**Comment by schris-dk** [comment]
Hi!

Currently we do not have a translator, but we would potentially present the option to make a cooperation with a client, that might want to join into this possibility.

Could you give an estimate of the number of pages to translate?

**Comment by evilaliv3** [comment]
The current strings are 600 and the total count of words is 3500.

A professional translator would probably take less of a day.
I suggest i could invite you to be coordinator for the Greenlandic language so that you could see the process of translations. With this role you will have the possibility to possibly add some translators and reviewers. Translators will be able to translate only what misses a translation and is not marked reviewed by a reviewer.

**Comment by schris-dk** [comment]
Hello Giovanni!
Regarding #3360 - Translation into Greenlandish. I do not currently have the names om the translators - and they are not the best 'tech savys'- which is why  I'd like to see the interface and see the procedure to help them out. Would this bepossible?
